Description: Removes makefile install of RXTXcomm.jar, silences an unnecessary compiling error
Origin: Mario Joussen <joussen@debian.org>, edited by Scott Howard <showard314@gmail.com> <2010-06-03>

Index: rxtx/src/RS485Imp.c
===================================================================
--- rxtx.orig/src/RS485Imp.c	2011-10-31 02:58:18.280985274 -0400
+++ rxtx/src/RS485Imp.c	2011-10-31 02:58:21.300985346 -0400
@@ -127,6 +127,7 @@
 #endif /* !__FreeBSD_kernel__ */
 #if defined(__linux__) 
 	/* Lets let people who upgraded kernels know they may have problems */
+	/*
 	if (uname (&name) == -1)
 	{
 		fprintf(stderr,"RXTX WARNING:  cannot get system name\n");
@@ -137,6 +138,7 @@
 		fprintf(stderr, "\n\n\nRXTX WARNING:  This library was compiled to run with OS release %s and you are currently running OS release %s.  In some cases this can be a problem.  Try recompiling RXTX if you notice strange behavior.  If you just compiled RXTX make sure /usr/include/linux is a symbolic link to the include files that came with the kernel source and not an older copy.\n\n\npress enter to continue\n",UTS_RELEASE,name.release);
 		getchar();
 	}
+	*/
 #endif /* __linux__ */
 #endif /* WIN32 */
 }
Index: rxtx/src/RawImp.c
===================================================================
--- rxtx.orig/src/RawImp.c	2011-10-31 02:58:18.280985274 -0400
+++ rxtx/src/RawImp.c	2011-10-31 02:58:21.300985346 -0400
@@ -272,6 +272,7 @@
 #endif /* !__FreeBSD_kernel__ */
 #if defined(__linux__) 
 	/* Lets let people who upgraded kernels know they may have problems */
+	/*
 	if (uname (&name) == -1)
 	{
 		fprintf(stderr,"RXTX WARNING:  cannot get system name\n");
@@ -282,6 +283,7 @@
 		fprintf(stderr, "\n\n\nRXTX WARNING:  This library was compiled to run with OS release %s and you are currently running OS release %s.  In some cases this can be a problem.  Try recompiling RXTX if you notice strange behavior.  If you just compiled RXTX make sure /usr/include/linux is a symbolic link to the include files that came with the kernel source and not an older copy.\n\n\npress enter to continue\n",UTS_RELEASE,name.release);
 		getchar();
 	}
+	*/
 #endif /* __linux__ */
 #endif /* WIN32 */
 }
Index: rxtx/src/SerialImp.c
===================================================================
--- rxtx.orig/src/SerialImp.c	2011-10-31 02:58:18.284985274 -0400
+++ rxtx/src/SerialImp.c	2011-10-31 02:58:21.304985346 -0400
@@ -320,6 +320,7 @@
 #endif /* DEBUG_TIMING */
 #if defined(DEBUG) && defined(__linux__)
 	/* Lets let people who upgraded kernels know they may have problems */
+	/*
 	if (uname (&name) == -1)
 	{
 		report( "RXTX WARNING:  cannot get system name\n" );
@@ -333,6 +334,7 @@
 		report( message );
 		getchar();
 	}
+	*/
 	LEAVE( "RXTXPort:Initialize" );
 #endif /* DEBUG && __linux__ */
 }
Index: rxtx/src/I2CImp.c
===================================================================
--- rxtx.orig/src/I2CImp.c	2011-10-31 02:58:18.284985274 -0400
+++ rxtx/src/I2CImp.c	2011-10-31 02:58:21.304985346 -0400
@@ -127,6 +127,7 @@
 #endif /* !__FreeBSD_kernel__ */
 #if defined(__linux__) 
 	/* Lets let people who upgraded kernels know they may have problems */
+	/*
 	if (uname (&name) == -1)
 	{
 		fprintf(stderr,"RXTX WARNING:  cannot get system name\n");
@@ -137,6 +138,7 @@
 		fprintf(stderr, "\n\n\nRXTX WARNING:  This library was compiled to run with OS release %s and you are currently running OS release %s.  In some cases this can be a problem.  Try recompiling RXTX if you notice strange behavior.  If you just compiled RXTX make sure /usr/include/linux is a symbolic link to the include files that came with the kernel source and not an older copy.\n\n\npress enter to continue\n",UTS_RELEASE,name.release);
 		getchar();
 	}
+	*/
 #endif /* __linux__ */
 #endif /* WIN32 */
 }
Index: rxtx/src/SerialImp.cpp
===================================================================
--- rxtx.orig/src/SerialImp.cpp	2011-10-31 02:58:18.284985274 -0400
+++ rxtx/src/SerialImp.cpp	2011-10-31 02:58:21.304985346 -0400
@@ -142,6 +142,7 @@
 #endif /* !__FreeBSD_kernel__ */
 #ifdef DEBUG
 	/* Lets let people who upgraded kernels know they may have problems */
+	/*
 	if (uname (&name) == -1)
 	{
 		report("RXTX WARNING:  cannot get system name\n");
@@ -153,6 +154,7 @@
 			name.release);
 		getchar();
 	}
+	*/
 #endif /* DEBUG */
 #endif /* __BEOS__ */
 #endif /* WIN32 */
Index: rxtx/Makefile.am
===================================================================
--- rxtx.orig/Makefile.am	2011-10-31 02:55:54.600981860 -0400
+++ rxtx/Makefile.am	2011-10-31 02:58:21.304985346 -0400
@@ -137,7 +137,8 @@
 
 #  create the javadoc files.
 docs: $(CLASSES)
-	$(JAVADOC) $(javafiles)
+	mkdir api
+	cd api && $(JAVADOC) $(javafiles)
 
 ################  WIN32 CrossCompiling from here down #######################
 
